the-collab-lab/tcl-44-smart-shopping-list
# Create header component

## Description
This PR creates a `Header` component that will be used in the three pages
- I Created a `Header` component and use it on our three pages.
- in the `Home` page it takes a `isHome` prop that will be used to style the header differently if we are in the home page.
- I changed the `Navbar` color to match the header and buttons' color.
- I moved the styles in `app.css` to `index.css`, I thought it's better to have only one stylsheet.



## Related Issue
closes #40 




## Type of Changes



|     | Type                       |
| --- | -------------------------- |
|    | :bug: Bug fix              |
|  ✓ | :sparkles: New feature     |
|    | :hammer: Refactoring       |
|    | :100: Add tests            |
|    | :link: Update dependencies |
|    | :scroll: Docs              |

## Updates

### Before
![before](https://user-images.githubusercontent.com/85396770/169837531-879d9fa6-b529-4f88-8e20-ea0313920e68.gif)




### After
![after](https://user-images.githubusercontent.com/85396770/169837566-38532d26-06f9-472b-bf56-04c1a0c65e64.gif)




## Testing Steps / QA Criteria
- From your terminal, pull down this branch with `git pull origin rb-create-header-component` and check that branch out with git checkout `b-create-header-component`.
- compare the changes with the Figma file.
